test/st001/test001.idr

```diff
@@ -0,0 +1,77 @@
+module Main 
+
+import Control.IOExcept
+import Control.ST
+import Control.ST.File
+import Control.ST.ImplicitCall
+
+increment : (x : Var) -> STrans m () [ x ::: State Int ]
+                                     (const [x ::: State Int])
+increment x = do num <- read x
+                 write x (num + 1)
+
+{-TestFileIO : Type -> (ConsoleIO m, STFile m) => ST m () []-}
+{-TestFileIO x = ?TestFileIO_rhs-}
```

Comment:
Why are these commented out?

Reply:
This is old code i forgot to remove. I will of course remove it.